aries-site/src/pages/components/menu.mdx

```diff
@@ -58,6 +60,19 @@ A menu can be used in a [Header](/components/header) to group related actions.
   <MenuHeaderExample />
 </Example>
 
+### Menu with Toolbar Button
+
+Use the toolbar button variant when it has to be placed in row with other toolbar controls like search or filter. 
```

Comment:
Can we add the sentence from https://design-system.hpe.design/components/button?q=button#toolbar-button that talks about the desire to give the buttons and input equal prominence?

Reply:
yes I can add that